src/app/services/dimDestinyTrackerService.factory.js

```diff
@@ -49,13 +49,16 @@ function DestinyTrackerService($q,
 
   return {
     reattachScoresFromCache: function(stores) {
-      if (_isDestinyOne()) {
-        _bulkFetcher.attachRankings(null,
+      if ((stores) &&
+          (stores[0])) {
```

Comment:
Why not follow the style of the rest of the DIM code? We don't use all these superfluous parentheses elsewhere.